docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '2.1'
+services:
+  docs:
+    build:
+      context: ./src/angularjs
+      dockerfile: Dockerfile
+    working_dir: /opt/angularjs
+    volumes:
+     - ./src/angularjs:/opt/angularjs
+     - ./src/angular/node_modules:/opt/angularjs/node_modules
```

Comment:
These lines are to keep development `node_modules` (which may not have been built on Linux) from colliding with the ones built during `cibuild`.